lib/src/source/hosted.dart

```diff
@@ -195,8 +195,8 @@ class HostedSource extends CachedSource {
   /// Returns a reference to a hosted package named [name].
   ///
   /// If [url] is passed, it's the URL of the pub server from which the package
-  /// should be downloaded. [url] most be normalized and validated using
-  /// [validateAndNormalizeHostedUrl].
+  /// should be downloaded. [url] will be normalized and validated using
+  /// [validateAndNormalizeHostedUrl]. This can throw a [FormatException].
```

Comment:
It probably meant to say "must" before.

Reply:
I'm guessing you changed the behavior, that's fine.

Reply:
yes